src/main/java/seedu/duke/logic/parser/exceptions/InvalidArgumentException.java

```diff
@@ -0,0 +1,8 @@
+package seedu.duke.logic.parser.exceptions;
+
+public class InvalidArgumentException extends ParserException {
+    public InvalidArgumentException() {
+        super("Invalid argument! Please follow the format for the command being used.\n"
```

Comment:
Should use `System.lineSeparator()` here? Idk we have` \n` in a lot of places tho..